### PR TITLE
Add Istio CNI configuration to Netd

### PR DESCRIPTION
**Pull request by dfloresh**

### CODE REVIEW

File: scripts/install-cni.sh [hunk 1]
```diff
@@ -121,6 +121,20 @@ else
...
 
+## Look for optional env var to know if plug-in should be enabled
+if [[ -n "${ISTIO_CNI_CONFIG:-}" ]]; then
+  ENABLE_ISTIO_PLUGIN=true
+fi
+
+# Add plug-in to config
+if [ "${ENABLE_ISTIO_PLUGIN}" == "true" ]; then
+  echo "Adding Istio plug-in to the CNI config."
+  cni_spec=${cni_spec//@cniIstioPlugin/, ${ISTIO_CNI_CONFIG}}
```
Comment (jingyuanliang) [inline scripts/install-cni.sh:132]: Add @cniIstioPlugin to netd-config in (the sample) netd.yaml.

File: scripts/install-cni.sh [hunk 2]
```diff
@@ -121,6 +121,20 @@ else
   cni_spec=${cni_spec//@cniCiliumPlugin/}
 fi
 
+## Look for optional env var to know if plug-in should be enabled
+if [[ -n "${ISTIO_CNI_CONFIG:-}" ]]; then
+  ENABLE_ISTIO_PLUGIN=true
```
Comment (jingyuanliang) [inline scripts/install-cni.sh:126]: I'm not sure if we'd better have this, or just use `-n "${ISTIO_CNI_CONFIG:-}"` whenever we want to check `"${ENABLE_ISTIO_PLUGIN}" == "true"`.

File: scripts/install-cni.sh (hunk 1 above)
Comment (dfloresh) [reply scripts/install-cni.sh:127]: Added @cniIstioPlugin.

**Review by howardjohn — COMMENTED** [review]
LGTM

**Pull request merged**